oximeter/db/src/client/mod.rs

```diff
@@ -1169,7 +1176,7 @@ impl Client {
                 self.log,
                 "failed to delete some timeseries";
                 "error" => ?error,
-                "call_count" => count,
+                "call_count" => ?count,
```

Comment:
Same question, just curious why this changed. Was this a problem before?

Reply:
same issue as above :)

Reply:
Sounds good, thanks.